Client/Application/AppDelegate.swift

```diff
@@ -110,47 +67,24 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         rootViewController = nil
     }
 
-    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
-        UIScrollView.doBadSwizzleStuff()
```

Comment:
The underlying issue this hack was added to fix has also been resolved.
https://bugs.webkit.org/show_bug.cgi?id=179628